code/_onclick/hud/hud.dm

```diff
@@ -82,6 +85,9 @@ GLOBAL_LIST_INIT(available_ui_styles, list(
 		plane_masters["[instance.plane]"] = instance
 		instance.backdrop(mymob)
 
+	screentip_text = new(FALSE, src)
```

Comment:
What's the FALSE for, mapload?

I'm like, nearly certain this is wrong

Reply:
its not being loaded in by a map

Reply:
What I mean is I'm nearly certain Initialize passes in mapload itself.

The first variable is the loc. You should just make this `null`.